src/calendar-list/index.tsx

```diff
@@ -108,13 +109,16 @@ const CalendarList = (props: CalendarListProps & ContextProp, ref: any) => {
 
   const [currentMonth, setCurrentMonth] = useState(parseDate(current));
 
+  const isAndroidRTL = useMemo(() =>constants.isRTL && constants.isAndroid && horizontal, [horizontal]);
```

Comment:
```suggestion
  const isAndroidRTL = useMemo(() => shouldUseAndroidRTLFix && horizontal, [horizontal]);
```
Need to take 'shouldUseAndroidRTLFix' from commons/constants